src/ExportModule.ts

```diff
@@ -32,10 +32,19 @@ export function exportModule(libName: string, dataTypes: CalculatorTypes.DataTyp
 
   dataTypes.forEach((object, i) => {
     const stateName = `${object.dataType}_${i}`;
+
     if (object.dataType !== null && factory(object.dataType, stateName) !== null) {
-      const StateClass = factory(object.dataType, stateName);
+      const stateClass = factory(object.dataType, stateName);
+      if (object.valueSet !== undefined && stateClass && states.doesHaveCodes(stateClass)) {
+        stateClass['codes'].push({
```

Comment:
```suggestion
        stateClass.codes.push({
```
Seems odd to use the bracket notation to access `codes` here.